src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs

```diff
@@ -88,6 +88,53 @@ public void StringIndexOfChar(int size)
             Assert.Equal(size / 2, index);
         }
 
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData(1, StringComparison.CurrentCulture)]
+        [InlineData(1, StringComparison.CurrentCultureIgnoreCase)]
+        [InlineData(1, StringComparison.InvariantCulture)]
+        [InlineData(1, StringComparison.InvariantCultureIgnoreCase)]
+        [InlineData(1, StringComparison.Ordinal)]
+        [InlineData(1, StringComparison.OrdinalIgnoreCase)]
+        [InlineData(10, StringComparison.CurrentCulture)]
+        [InlineData(10, StringComparison.CurrentCultureIgnoreCase)]
+        [InlineData(10, StringComparison.InvariantCulture)]
+        [InlineData(10, StringComparison.InvariantCultureIgnoreCase)]
+        [InlineData(10, StringComparison.Ordinal)]
+        [InlineData(10, StringComparison.OrdinalIgnoreCase)]
+        [InlineData(100, StringComparison.CurrentCulture)]
+        [InlineData(100, StringComparison.CurrentCultureIgnoreCase)]
+        [InlineData(100, StringComparison.InvariantCulture)]
+        [InlineData(100, StringComparison.InvariantCultureIgnoreCase)]
+        [InlineData(100, StringComparison.Ordinal)]
+        [InlineData(100, StringComparison.OrdinalIgnoreCase)]
+        [InlineData(1000, StringComparison.CurrentCulture)]
+        [InlineData(1000, StringComparison.CurrentCultureIgnoreCase)]
+        [InlineData(1000, StringComparison.InvariantCulture)]
+        [InlineData(1000, StringComparison.InvariantCultureIgnoreCase)]
+        [InlineData(1000, StringComparison.Ordinal)]
+        [InlineData(1000, StringComparison.OrdinalIgnoreCase)]
+        public void IndexOf_StringComparison(int size, StringComparison comparisonType)
+        {
+            string str = new string('0', size / 2) + "5";
```

Comment:
This is a good baseline test, but we should also consider measuring scenarios with culture specific characters (not only ascii) and upper and lower case characters together.

Reply:
I can not run any performance test for netfx ;(